hack/bootstrap/create/main.go

```diff
@@ -175,7 +177,7 @@ func deployDir() string {
 func main() {
 	flag.Parse()
 	config.bind(*mode)
-	preloadImages()
+	// preloadImages()
```

Comment:
Do we want to remove `preloadImages` at the moment? Maybe we can leave it to separate PR?